api/ServiceWorkerRegistration.json

```diff
@@ -0,0 +1,704 @@
+{
+  "api": {
+    "ServiceWorkerRegistration": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerRegistration",
+        "support": {
+          "webview_android": {
+            "version_added": "40"
+          },
+          "chrome": {
+            "version_added": "40"
+          },
+          "chrome_android": {
+            "version_added": "40"
+          },
+          "edge": {
+            "version_added": "17",
+            "notes": "Service workers is available in Microsoft Edge starting EdgeHTML 16 behind a flag."
```

Comment:
This needs a "flags" property and likely a bit of digging to discover what flag is needed.

Reply:
So likely:
[ { 
    "version_added": "17"
   },
   {
    "version_added: "16",
    "flags": { <em>something</em> }
   }
]